scaffold/github/actions/common/ddev/action.yml

```diff
@@ -60,6 +62,14 @@ runs:
           sudo apt install -y ddev
         fi
 
+        if [ "${{ inputs.replace-ddev-binary-url }}" != "" ]; then
+          echo "Replacing ddev binary from ${{ inputs.replace-ddev-binary-url }}"
+          curl --silent --location --output /tmp/ddev.zip "${{ inputs.replace-ddev-binary-url }}"
+          sudo unzip -o -d /usr/bin /tmp/ddev.zip
+          sudo chmod +x /usr/bin/ddev
+          ddev --version
+        fi
+
```

Comment:
Since I was able to reproduce this so easily in this PR, we won't want to merge this whole PR. @justafish what do you think about adding support for overriding ddev like this as a feature?

Also Randy says:

> Note that you're using an obsolete technique for key install, Run curl https://apt.fury.io/drud/gpg.key | sudo apt-key add -
> Also, you're using the old drud location, https://apt.fury.io/drud

Reply:
@deviantintegral sounds good! :+1:
https://github.com/Lullabot/drainpipe/issues/528